src/web/app/components/sessions-table/send-reminders-to-respondents-modal/send-reminders-to-respondents-modal.component.ts

```diff
@@ -64,10 +67,17 @@ export class SendRemindersToRespondentsModalComponent {
     });
   }
 
+  /**
+   * Changes selection state for sending a copy to requesting instructor.
+   */
+  changeSelectionStatusForSendingCopyToInstructorHandler(shouldSelect: boolean): void {
+    this.isSendingCopyToInstructor = shouldSelect;
+  }
```

Comment:
`shouldSendCopy` will be a better param name here. `shouldSelect` makes sense for the other change handlers (because we are selecting students), but not this one.